superdesk/io/feed_parsers/__init__.py

```diff
@@ -0,0 +1,173 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from abc import ABCMeta, abstractmethod
+
+from superdesk.etree import etree
+from superdesk.locators.locators import find_cities
+from superdesk.metadata.item import Priority
+
+
+class FeedParser(metaclass=ABCMeta):
```

Comment:
OK. Can we do the same thing for `can_parse()` or is it ok how it is now in this pr?